lvgl_tft/Kconfig

```diff
@@ -469,14 +469,15 @@ menu "LVGL TFT Display controller"
 
     choice
         prompt "TFT SPI Bus." if LV_TFT_DISPLAY_PROTOCOL_SPI
-        default LV_TFT_DISPLAY_SPI2_HOST
-        help
-              Select the SPI Bus the TFT Display is attached to.
+        default LV_TFT_DISPLAY_SPI2_HOST if LV_PREDEFINED_DISPLAY_TTGO && !IDF_TARGET_ESP32S2
+        default LV_TFT_DISPLAY_SPI3_HOST if !LV_PREDEFINED_DISPLAY_TTGO && IDF_TARGET_ESP32S2
```

Comment:
Why do chose exactly SPI3_HOST for ESP32S2 ?

Reply:
This PR is based on a patch received on the linked comment. Do you think it would be better to choose another SPI? I'm not familiar with that particular device

Reply:
As of datasheet and online docs esp32s2 do have two working general purpose SPI: SPI2 and SPI3.  I'm just wondering why SPI3_HOST is the default choice for esp32s2. Also TTGO T Display board have esp32 (not esp32s2), that's why `!LV_PREDEFINED_DISPLAY_TTGO && IDF_TARGET_ESP32S2` should be always true, unless someone is purposely picking esp32s2 as target, but pin configuration is hardcoded and cannot be changed  if `LV_PREDEFINED_DISPLAY_TTGO` is picked.

Reply:
What is the purpose of this code anyway?

Reply:
I can remove this change and leave only the fix on `LV_TFT_DISPLAY_X_OFFSET` and `LV_TFT_DISPLAY_Y_OFFSET` configs. What do you think?

Reply:
I think `&& !IDF_TARGET_ESP32S2` part should be removed at least.